pom.xml

```diff
@@ -99,6 +99,7 @@
     <netty-http.version>0.16.0</netty-http.version>
     <commons-math3.version>3.6.1</commons-math3.version>
     <slf4j.version>1.7.5</slf4j.version>
+    <geogson.version>1.1.97</geogson.version>
```

Comment:
The library is called geogson, it provides to parse the geojson fomat into gson objects
https://github.com/filosganga/geogson